src/screens/Main/MainScreen.navigation.tsx

```diff
@@ -0,0 +1,47 @@
+import 'react-native-gesture-handler';
```

Comment:
I checked with only App.tsx import (removed it from navigation files) and everything works for me